src/core/session.c

```diff
@@ -1111,6 +1111,7 @@ QuicSessionParamSet(
 
     case QUIC_PARAM_SESSION_SERVER_RESUMPTION_LEVEL: {
         if (BufferLength != sizeof(QUIC_SERVER_RESUMPTION_LEVEL) ||
+            Buffer == NULL ||
```

Comment:
If this check is needed, it should be at a higher level. I thought the function that calls this already does this check.

Reply:
No, it appears this is left to each parameter to check individually. Which makes some sense because some parameters may want to use NULL as a sentinel value.